src/runway.rs

```diff
@@ -115,6 +122,11 @@ where
     rx_consensus: Receiver<NotificationOut<H>>,
     ordered_batch_rx: Receiver<Vec<H::Hash>>,
     data_io: DP,
+    after_catch_up_delay: bool,
+    starting_round_sender: Option<oneshot::Sender<Round>>,
```

Comment:
I hate that all this data that is only needed during setup is part of the struct forever. I don't think we can expect to fix this now, since it might require a larger rewrite that we don't necessarily have time for, but we at least need an issue for that.

Reply:
Added a task in Jira